### ISSUE TITLE
Sentry is a security / data protection risk

### ISSUE THREAD

**Issue by lukestanley** [Paxa/postbird#70]
Postbird seems pretty great but debug logs probably shouldn't be sent without explicit consent, especially considering they may be expected to contain sensitive data or connection info. (Even if Sentry claim to scrub it, it's still sent out.)
This is a massive data protection / GDPR / privacy risk to Postbird users.
By default, sensitive data probably should not be leaked.
Consent should probably be gathered and stored.

**Comment by Paxa** [comment]
You are right, I should ask user confirmation before sending. Will fix it in next version

Currently it send computer name, system username, system version, postgres server version, user IP (I don't know how to turn if off) and stack trace.
Database credentials are not sent to sentry

I guess all of that is not personal, what do you think?

![screencapture-sentry-io-organizations-postbird-issues-628620087-events-latest-2019-06-27-00_42_12](https://user-images.githubusercontent.com/26019/60198638-a85ff500-9874-11e9-90f5-be1638b64df6.png)

**Comment by lukestanley** [comment]
Thanks for acknowledging it and committing to a fix, that's excellent! 👍 

My understanding is that under GDPR, user names and IP's are typically considered potentially sensitive.

Regarding the contents of the data sent:
While traceback may not have the connection string's manually added to it, I don't see anything preventing them from being sent up as a natural part of a traceback.
For example, even if there was a connection string parsing traceback, without doing a HTTPS MITM to check, it might not be obvious what it's sending but it sends up a lot for sure.
